components/gc-chckbxrdio/gc-chckbxrdio-doc-en.html

```diff
@@ -0,0 +1,176 @@
+---
+title: Checkboxes and radio buttons
+language: en
+tag: chckbxrdio
+category: Pattern
+parentdir: components
+description: Checkboxes and radio buttons
+dateModified: 2021-05-07
+altLangPage: gc-chckbxrdio-doc-fr.html
+---
+
+<span class="wb-prettify all-pre hide"></span>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Stable</dd>
+	<dt>Type</dt>
+	<dd>Canada.ca design pattern</dd>
+	<dt>Last review</dt>
+	<dd>2021-05-07</dd>
+	<dt>Conforming to</dt>
+	<dd>Content and IA spec 2.1</dd>
```

Comment:
This is not defined by the content and IA spec, but just by the "Template and design pattern library" which is not versioned currently.